### PR TITLE
Game teams manager refactor

### PR DESCRIPTION
**Pull request by cpfergus1**
Going to merge with reorg branch after fixing bugs

### CODE REVIEW

File: lib/game_team_manager.rb [hunk 1]
```diff
@@ -19,12 +21,9 @@ def generate_game_teams(location) # I need a test
   def goal_avg_per_team(team_id, home_away)
     goal_array = []
     @game_teams.each do |game|
-        if game.team_id == team_id && home_away == game.HoA
```
Comment (s-steel) [inline lib/game_team_manager.rb:22]: Nice refactor

File: lib/game_team_manager.rb [hunk 2]
```diff
@@ -71,32 +68,20 @@ def game_team_info(game_id)
...
 
-  def team_by_id(team_id)
-    @stat_tracker.team_info(team_id)['team_name']
-  end
-
   def game_teams_data_for_season(season_id)
-    @game_teams.find_all do |game|
-      game.game_id[0..3] == season_id[0..3]
-    end
+    @game_teams.find_all { |game| game.game_id[0..3] == season_id[0..3] }
```
Comment (s-steel) [inline lib/game_team_manager.rb:72]: This looks a lot cleaner.

File: lib/shared_calc.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# methods used in multiple places
```
Comment (s-steel) [inline lib/shared_calc.rb:1]: I like having this module available.

**Review by s-steel — APPROVED** [review]
I like it.  Nice refactoring and I like the implementation of the module.

**Review by c-bartell — APPROVED** [review]
The refactors look great!

**Review by s-steel — APPROVED** [review]
Looks good, nice work

**Pull request merged**